Formula/cloudflare-quiche.rb

```diff
@@ -21,10 +21,22 @@ class CloudflareQuiche < Formula
   depends_on "rust" => :build
 
   def install
+    system "cargo", "build", "--features", "ffi"
```

Comment:
I'm getting this error when passing in `--locked`: 

"error: the lock file /private/tmp/cloudflare-quiche-20230518-21451-1o84y5o/Cargo.lock needs to be updated but --locked was passed to prevent this
If you want to try to generate the lock file without accessing the network, remove the --locked flag and use --offline instead." 

Could I remove this and use `--offline` instead?

Reply:
The repo appears to not have a `Cargo.lock` file, which is why `cargo build --locked` doesn't work.

Not sure why `cargo install --locked` works, though. This is from the `cargo-install` manpage:
```
       --frozen, --locked
           Either of these flags requires that the Cargo.lock file is up-to-date. If the lock file is missing, or it needs to be
           updated, Cargo will exit with an error. The --frozen flag also prevents Cargo from attempting to access the network to
           determine if it is out-of-date.

           These may be used in environments where you want to assert that the Cargo.lock file is up-to-date (such as a CI build) or
           want to avoid network access.
```
I've noticed this before in other formulae though. I suspect this is just a bug in `cargo`. (Or a bug in the documentation.)

Reply:
@marshallstone, if you had the time to report the inconsistency between the behaviour of `cargo install` and the documentation quoted above to the cargo project, that'd be much appreciated.

Reply:
I could probably take a look at some point :)
- thanks for all the help!